tests/test_crypto.py

```diff
@@ -588,55 +578,50 @@ def test_construction(self):
         :py:class:`X509ExtensionType` instance.
         """
         basic = X509Extension(b'basicConstraints', True, b'CA:true')
-        self.assertTrue(
-            isinstance(basic, X509ExtensionType),
-            "%r is of type %r, should be %r" % (
-                basic, type(basic), X509ExtensionType))
+        assert isinstance(basic, X509ExtensionType)
```

Comment:
Note: I dropped the custom error message because pytest is smart enough to include information about the actual type of `basic` without it; for example:

```
__________________________________ TestX509Ext.test_construction ___________________________________

self = <tests.test_crypto.TestX509Ext object at 0x105e0d5c0>

    def test_construction(self):
        """
            :py:class:`X509Extension` accepts an extension type name, a critical
            flag, and an extension value and returns an
            :py:class:`X509ExtensionType` instance.
            """
        basic = X509Extension(b'basicConstraints', True, b'CA:true')
>       assert isinstance(basic, ValueError)
E       assert False
E        +  where False = isinstance(<OpenSSL.crypto.X509Extension object at 0x105e0d780>, ValueError)

tests/test_crypto.py:590: AssertionError
```